src/test/unittest/basetest.py

```diff
@@ -168,6 +168,22 @@ def _get_utenv(self):
             'UNITTEST_NUM': str(self.testnum)
         }
 
+    def _debug_log_env(self):
+        """
+        Return environment variables that enable logging PMDK debug output
+        into log files
+        """
+        libs = ('pmem', 'pmem2', 'pmemobj', 'pmemblk', 'pmemlog', 'pmempool')
+
+        envs = {}
+        for l in libs:
+            envs['{}_LOG_LEVEL'.format(l.upper())] = '3'
+            log_file = os.path.join(self.cwd,
+                                    '{}{}.log'.format(l, self.testnum))
+            envs['{}_LOG_FILE'.format(l.upper())] = log_file
```

Comment:
Please use it -> rm ` ctx.env['PMEM2_LOG_FILE'] = self.pmem2_log` from `pmem2_mem_ext/TESTS.py`